content/docs/command-reference/commit.md

```diff
@@ -19,9 +19,9 @@ positional arguments:
 
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
-[pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force update the
-`dvc.lock` or `.dvc` files without reproducing stages or pipelines. These
+[pipeline](/doc/command-reference/dag) is in development/experimentation; to
+force-update the `dvc.lock` or `.dvc` files without reproducing stages or
+pipelines; or to add missing outputs or manually execute a stage. These
```

Comment:
> or to add missing outputs or manually execute a stage.

Yes, definitely simpler which is good. But this is still 2 separate things (so should use a comma, removing a coma is not = merging). This should be something like "or to mark existing files/dirs as stage outputs" — a single generalization that covers the last 2 bullets.

Also, `abbr` is needed around "outputs".

Reply:
Done.